dev/bots/test.dart

```diff
@@ -140,13 +144,31 @@ Future<void> _runSmokeTests() async {
   await _verifyVersion(path.join(flutterRoot, 'version'));
 }
 
+Future<bq.BigqueryApi> _getBigqueryApi() async {
+  // TODO(dnfield): How will we do this on LUCI?
```

Comment:
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-checking-in-comments-that-ask-questions